package.json

```diff
@@ -7,6 +7,7 @@
     "npm": ">= 5.3.0"
   },
   "main": "lib/index.js",
+  "publishConfig": { "registry": "https://npm.pkg.github.com/" },
```

Comment:
@talsalmona no need for this change

Reply:
Sorry, was unintentional. I'll remove it.